great_expectations/render/renderer/content_block/content_block.py

```diff
@@ -23,10 +25,10 @@
 
 
 class ContentBlockRenderer(Renderer):
-    _rendered_component_type = TextContent
+    _rendered_component_type: Type[RenderedComponentContent] = TextContent
     _default_header = ""
 
-    _default_content_block_styling = {"classes": ["col-12"]}
+    _default_content_block_styling: Dict[str, JSONValues] = {"classes": ["col-12"]}
```

Comment:
Just a question to help me understand better - is the reason that this is typed with `Dict[str, JSONValues]` and not just `JSONValues` because we know that it has str keys at the top level?